container/sagemaker/python_service.py

```diff
@@ -215,6 +246,60 @@ def _read_model_config(self):
                 line = f.readline()
         return models
 
+    def validate_model_dir(self, model_path):
+        # model base path doesn't exits
+        if not os.path.exists(model_path):
+            return False
+        # model versions doesn't exist
+        versions = []
+        for _, dirs, _ in os.walk(model_path):
+            for dirname in dirs:
+                if dirname.isdigit():
+                    versions.append(dirname)
+        return bool(versions)
```

Comment:
I'm confused. How does this work? Why are we converting the version to a bool?

Reply:
This validation method checks if the model file has the correct structure:
1. the model dir exists
2. the model dir contains at least 1 version (aka at least 1 sub folder with version number)

bool(versions) returns False with an empty version list, otherwise it returns True

Reply:
Ah, gotcha. Should we make it more obvious what this does by doing something like this?
```
if versions:
    return True
return False
```
_Non-blocking comment._

Reply:
Done :)